src/app/features/home/onboarding/prefetching-dialog/prefetching-dialog.component.ts

```diff
@@ -26,6 +30,12 @@ export class PrefetchingDialogComponent {
           (this.progress = currentCount / totalCount)
       );
       await this.onboardingService.setHasPrefetchedDiaBackendAssets(true);
+    } catch {
+      this.dialogRef.close();
+      await this.confirmAlert.present({
```

Comment:
Showing the confirm alert with two buttons (Cancel and OK) might confused the user. They don't have any difference in this case, but the user will be uncertain of that.